x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/PlanExecutor.java

```diff
@@ -50,14 +50,16 @@ public PlanExecutor(Client client, IndexResolver indexResolver, NamedWriteableRe
         this.preAnalyzer = new PreAnalyzer();
         this.optimizer = new Optimizer();
         this.planner = new Planner();
+        
+        this.verifier = new Verifier();
```

Comment:
The `Verifier` is used before the `Optimizer` so declare it as such including in the following method declarations.